arm-appinsights/2015-05-01/swagger/syntheticMonitor_API.json

```diff
@@ -0,0 +1,402 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AppInsightsManagementClient",
+    "description": "Azure Application Insights client for synthetic monitor based alerting.",
+    "version": "2015-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/webtests": {
+      "get": {
+        "description": "Get all Application Insights synthetic monitors defined within a specified resource group.",
+        "operationId": "SyntheticMonitor_ListByResourceGroup",
```

Comment:
This will cause linter violations. We have a recommendation from the ARM team to name resource types the same as they are in the Url. Different names causes confusion.
@ravbhatnagar @salameer @veronicagg - please take a note of this.